scripts/coords.js

```diff
@@ -0,0 +1,70 @@
+
+var _ = require('./constants.js');
```

Comment:
Because `_` usually means a reference to either the underscore or lodash libraries, let's call this `config`.